lib/email/bounces.js

```diff
@@ -52,56 +45,66 @@ module.exports = function (log, error) {
 
     function handleBounce(message) {
       var recipients = []
-      if (message.bounce && message.bounce.bounceType === 'Permanent') {
```

Comment:
> would prefer to tackle it as a separate issue rather than conflating with this feature.

To clarify, the action suggested is that in this PR, put a check in so we still only delete if bounce type was permanent?